src/serialmonitor/serialportctrl.ts

```diff
@@ -93,7 +96,7 @@ export class SerialPortCtrl {
 
     return new Promise((resolve, reject) => {
         this._child = spawn(SerialPortCtrl._serialCliPath,
-            ["open", this._currentPort, "-b", this._currentBaudRate.toString(), "--json"])
+            ["open", this._currentPort, "-b", this._currentBaudRate.toString(), "-t", this._currentTimestampFormat, "--json"])
 
         this._child.on("error", (err) => {
             reject(err)
```

Comment:
After the timestamp is changed to be a field of the json object, you'll want to change lines 105-108 (inside the child.stdou.on("data"...)) to reflect that.

Reply:
Resolved in https://github.com/microsoft/vscode-arduino/pull/1430/commits/871c88c3f699316a3eec982344f9772ea693bd67